src/pytorch_lightning/callbacks/progress/progress.py

```diff
@@ -0,0 +1,291 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Dict, Optional, Union
+
+import pytorch_lightning as pl
```

Comment:
Everything is supposed to be:

`import lightning as L`

and then `L.Trainer`, `L.LightningModule`, and `L.LightningDataModule`

Reply:
Thank you for your review. I'm very confused because this is the standalone PL package but not the lightning meta package cc @Borda

Reply:
Yes, @Felonious-Spellfire that is for documentation, but this is the code inside the `pytorch_lightning` codebase